.github/workflows/test.yml

```diff
@@ -14,6 +14,9 @@ jobs:
         go-version: ${{ matrix.go-version }}
     - name: Checkout code
       uses: actions/checkout@v4
+    - if: ${{ matrix.go-version == '1.18.x' }}
+      run: |
+        echo 'PACKAGES="mountinfo mount sequential signal symlink user"' >> $GITHUB_ENV
```

Comment:
This is the same list as `Makefile:1`, but minus `userns`, right?  That's somewhat clear while reviewing this diff while they're close together, but will no longer be clear the minute we merge this. :sweat_smile:  How do we make sure we remember to update this list in the future?  (and know how to update it correctly?)

Reply:
Yeah, no good solution for that, honestly. Unless we probably introduce more complexity.

Overall, the risk is low;

- if we don't update, the default is still all packages from the Makefile
- if a new module is added that doesn't support go1.18, that module will thus fail
- if we add a new module that should support go1.18, but we forgot to update; that module is skipped in the go1.18 tests (but still tested on current versions)


I'd consider only the last one slightly problematic, but perhaps a calculated risk, given that adding more modules should be a rare event 🙈

Reply:
erm: middle bullet is obviously wrong; so only 1st and last apply 🤦‍♂️